### ISSUE TITLE
python2-clustershell RPM not being generated

### ISSUE THREAD

**Issue by mattaezell** [cea-hpc/clustershell#445]
After cdbb5ef (@e4t) I'm not getting `python2-clustershell` RPMs on RHEL7 or SLES15. I'm not an expert in RPM logic, but it seems that removing the negate and flipping the equality fixes the issue:
```
diff --git a/clustershell.spec.in b/clustershell.spec.in
index ed25732..6cb97ef 100644
--- a/clustershell.spec.in
+++ b/clustershell.spec.in
@@ -21,7 +21,7 @@
 %{!?__python3: %global __python3 python3}
 %{!?python3_shortver: %global python3_shortver %(%{__python3} -c 'import sys; print(str(sys.version_info.major) + "." + str(sys.version_info.minor))')}
 
-%if ! 0%{?rhel} >= 8 && ! 0%{?suse_version} > 1550
+%if 0%{?rhel} < 8 && 0%{?suse_version} <= 1550
 %define py2 1
 %endif

```

It seems that this disabled python2 on RHEL8, but I believe it actually supports both. I haven't tried to build clustershell on EL8, what's it supposed to do?

**Comment by degremont** [comment]
@e4t or @thiell ? Any idea on this?

**Comment by e4t** [comment]
Indeed. I was trying to avoid the issue with `0%{?..} < something` which did not really work anyway - and I've missed to enclose the test in `(..)`.
The suggested logic does exactly what is needed. Will submit an update.

**Comment by e4t** [comment]
Fix is pull-requested, now. 
About Python2 support in RHEL8:
According to https://developers.redhat.com/blog/2018/11/14/python-in-rhel-8/ python3 is now the default while python2 remains available.
I've followed the logic that had existed in the spec file before regarding RHEL.
Going forward, it is probably the right thing to drop Python2 - RHEL 8 has been released later than SLE 15 - which will be the last SLE with python2 support.
